### PR TITLE
[INLONG-5701][Manager][Sort] Support raw format

### LINKED ISSUE #5701: [Feature][Manager][Sort] Support raw format
### Description

Support raw format

### Use case

none

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
### Prepare a Pull Request

[INLONG-5701][Manager][Sort] Support raw format

- Fixes #5701 

### Motivation

- Pulsar and Kafka source support set raw format.

### Modifications

- Add raw format

### CODE REVIEW

File: inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/RawFormat.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonTypeName;
+
+/**
+ * The Raw format
+ *
+ * @see <a herf="https://nightlies.apache.org/flink/flink-docs-release-1.13/zh/docs/connectors/table/formats/raw/">
+ *         Raw Format</a>
+ */
+
```
Comment (healchow) [inline inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/RawFormat.java:35]: Unnecessary blank line.

Comment (e-mhui) [reply inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/format/RawFormat.java]: done

**Review by yunqingmoswu — APPROVED** [review]
LGTM

**Review by healchow — APPROVED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java [hunk 2]
```diff
@@ -181,6 +182,9 @@ public static KafkaExtractNode createExtractNode(KafkaSource kafkaSource) {
             case DEBEZIUM_JSON:
                 format = new DebeziumJsonFormat();
                 break;
+            case RAW:
+                format = new RawFormat();
+                break;
```
Comment (gong) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java:187]: Maybe, you need modify tableOptions of `KafkaExtractNode` if you add this format.

Comment (e-mhui) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java]: Done, it has been removed.

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**